Building the Matlab interface fails

**Problem description**

With the current `main` (https://github.com/Cantera/cantera/commit/643fdea8208d619d5168f4d66f80f684ca2d876d) branch, building the Matlab interface fails:

```
src/matlab/flowdevicemethods.cpp:77:21: error: no matching function for call to 'flowdev_massFlowRate'
                r = flowdev_massFlowRate(i, v);
                    ^~~~~~~~~~~~~~~~~~~~
include/cantera/clib/ctreactor.h:57:25: note: candidate function not viable: requires single argument 'i', but 2 arguments were provided
    CANTERA_CAPI double flowdev_massFlowRate(int i);
```

**Steps to reproduce**

1. `git switch main && scons build matlab_toolbox=y`

- OS: macOS 10.14 with the Apple clang compiler

**Additional context**

https://github.com/Cantera/conda-recipes/runs/1927627896#step:10:1212

Cantera/conda-recipes#24

I can reproduce the error on a windows build using VS19 ([see setup](https://gist.github.com/ischoegl/ab777f871a48d7f40ecc32c1c849faa3)) at the `scons build` stage:
```
[...]
flowdevicemethods.cpp 
src\matlab\flowdevicemethods.cpp(77): error C2660: 'flowdev_massFlowRate': function does not take 2 arguments
C:\Users\ischo\GitHub\cantera\include\cantera/clib/ctreactor.h(57): note: see declaration of 'flowdev_massFlowRate'
scons: *** [build\src\matlab\flowdevicemethods.obj] Error 2
scons: building terminated because of errors.
```
**PS:** Adding a reference to #977 here (specifically abff4359096df92bee49cb5a2051f348b4fa095e)

--- a/src/matlab/flowdevicemethods.cpp
+++ b/src/matlab/flowdevicemethods.cpp
@@ -71,11 +71,7 @@ void flowdevicemethods(int nlhs, mxArray* plhs[],
         // options that return a value of type 'double'
         switch (job) {
         case 21:
-            if (v == Undef) {
-                r = flowdev_massFlowRate2(i);
-            } else {
-                r = flowdev_massFlowRate(i, v);
-            }
+            r = flowdev_massFlowRate(i);
             break;
         default:
             mexErrMsgTxt("unknown job parameter");

--- a/src/matlab/flowdevicemethods.cpp
+++ b/src/matlab/flowdevicemethods.cpp
@@ -20,7 +20,7 @@ void flowdevicemethods(int nlhs, mxArray* plhs[],
     if (job == 0) {
         char* type = getString(prhs[2]);
 
-        n = flowdev_new2(type);
+        n = flowdev_new(type);
         plhs[0] = mxCreateNumericMatrix(1,1,mxDOUBLE_CLASS,mxREAL);
         double* h = mxGetPr(plhs[0]);
         *h = double(n);

--- a/src/matlab/reactormethods.cpp
+++ b/src/matlab/reactormethods.cpp
@@ -24,7 +24,7 @@ void reactormethods(int nlhs, mxArray* plhs[],
     if (job == 0) {
         char* type = getString(prhs[2]);
 
-        n = reactor_new2(type);
+        n = reactor_new(type);
         plhs[0] = mxCreateNumericMatrix(1,1,mxDOUBLE_CLASS,mxREAL);
         double* h = mxGetPr(plhs[0]);
         *h = double(n);

--- a/src/matlab/wallmethods.cpp
+++ b/src/matlab/wallmethods.cpp
@@ -23,7 +23,7 @@ void wallmethods(int nlhs, mxArray* plhs[],
     // constructor
     if (job == 0) {
         char* type = getString(prhs[2]);
-        n = wall_new2(type);
+        n = wall_new(type);
         plhs[0] = mxCreateNumericMatrix(1,1,mxDOUBLE_CLASS,mxREAL);
         double* h = mxGetPr(plhs[0]);
         *h = double(n);

